app/assets/javascripts/views/story_view.js

```diff
@@ -75,20 +75,7 @@ module.exports = FormView.extend({
     // Set up CSS classes for the view
     this.setClassName();
 
-    this.attachinaryOptions = {}
-    this.fetchAttachinaryOptions();
-  },
-
-  fetchAttachinaryOptions: function() {
-    const attachmentOptions = new AttachmentOptions({
-      refreshCallback: this.setAttachinaryOptions.bind(this)
-    });
-    attachmentOptions.fetch();
-  },
-
-  setAttachinaryOptions: function(options) {
-    this.attachinaryOptions = options;
-    this.render();
+    this.attachinaryOptions = window.attachinarySignature
```

Comment:
Will this be refreshed when the signature is fetched again? Is it what happens here https://github.com/Codeminer42/cm42-central/blob/4588c860155d2c6a44b6db49e135f63e7ecfc0d2/app/assets/javascripts/views/story_view.js#L520 ?

Reply:
I think I answer this in chat, sorry I'm late :(